config/tests/testgrids/BUILD.bazel

```diff
@@ -0,0 +1,47 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_test")
+
+go_test(
+    name = "go_default_test",
+    srcs = ["config_test.go"],
+    args = ["--config=$(location testconf.pb)", "--prow-config=$(location //prow:config.yaml)", "--job-config=config/jobs"],
```

Comment:
I'd be inclined to put these all on their own lines

Reply:
Fixed in revision